zh-CN/mapping_dev.zh_CN

```diff
@@ -48,95 +48,95 @@ dlgProgress.tle"JIRA"=JIRA
 dlgProgress.tle"Jump To"=跳转到
 dlgProgress.tle"Refresh"=刷新
 dlgProgress.tle"Reveal Commit"=显示提交
-dlgProgress.tle"SmartGit Installation Update"=SmartGit安装更新
-dlgQBugFileSendingFailed.fur%1=也许您需要配置代理才能访问我们服务器或我们的服务器暂时关闭.\n详情: $1
-dlgQBugFileSendingFailed.hdl%1=无法将崩溃日志发送到$1
+dlgProgress.tle"SmartGit Installation Update"=SmartGit 安装更新
+dlgQBugFileSendingFailed.fur%1=也许您需要配置代理才能访问我们服务器或我们的服务器暂时关闭。\n详情：$1
+dlgQBugFileSendingFailed.hdl%1=无法将崩溃日志发送到 $1
 dlgQBugFileSendingFailed.tle=本机崩溃日志
 dlgQBugReportSend.btn"Force Exit"=强制退出
-dlgQBugReportSend.btn"Report Bug"=反馈Bug
+dlgQBugReportSend.btn"Report Bug"=反馈 Bug
 dlgQBugReportSend.edt"Comments or Steps to Reproduce"=重现注释或步骤
 dlgQBugReportSend.edt"Email"=电子邮箱
 dlgQBugReportSend.tle=内部错误
 dlgQDockManagerClosedView.chk=不再显示
-dlgQDockManagerClosedView.fur=要再次重新打开它,请使用窗口菜单中的相应菜单项。
-dlgQDockManagerClosedView.hdl%1=You've closed the view '$1'.
-dlgQDockManagerClosedView.tle=Closed View
+dlgQDockManagerClosedView.fur=要重新打开它，请使用窗口菜单中相应的菜单项。
+dlgQDockManagerClosedView.hdl%1=你已经关闭了视图 $1。
+dlgQDockManagerClosedView.tle=关闭视图
 dlgQFileSaveAcceptFilterOverwrite.btn"Overwrite"=覆盖
-dlgQFileSaveAcceptFilterOverwrite.fur=要保存到其他文件，请单击“取消”。
-dlgQFileSaveAcceptFilterOverwrite.hdl%1=文件$1已存在。你确定覆盖它吗？
+dlgQFileSaveAcceptFilterOverwrite.fur=要保存到其他文件，请单击取消。
+dlgQFileSaveAcceptFilterOverwrite.hdl%1=文件 $1 已存在。你确定覆盖它吗？
 dlgQFileSaveAcceptFilterOverwrite.tle=覆盖文件
-dlgQIntegerInputProviderInvalidValue.fur%2=端口必须在$1到$2的范围内。
-dlgQIntegerInputProviderInvalidValue.hdl%1=字段“$1”中的文本无效。
+dlgQIntegerInputProviderInvalidValue.fur%2=端口必须在 $1 到 $2 的范围内。
+dlgQIntegerInputProviderInvalidValue.hdl%1=字段 $1 中的文本无效。
 dlgQIntegerInputProviderInvalidValue.tle=输入验证
 dlgQProxyConfigure.chk"Proxy requires authentication"=代理需要身份验证
 dlgQProxyConfigure.edt"Host"=服务器
 dlgQProxyConfigure.edt"Password"=密码
 dlgQProxyConfigure.edt"Port"=端口
 dlgQProxyConfigure.edt"Username"=用户名
 dlgQProxyConfigure.lbl"Note: The password will be stored in plain text in SmartGit's configuration area!"=\
- 注意：密码将以纯文本形式存储在SmartGit的配置区域中！
+ 注意：密码将以明文形式存储在 SmartGit 的配置区域中！
```

Comment:
I have seen many places deliberately separate English by spaces, and some places are needed (for example: about SmartGit), but most places are not needed(E.g this place), plus it will be strange.

I will not point out one by one in other places, please confirm and update.

Reply:
When I translate, I refer to the typesetting of this guide to a certain extent. https://github.com/taotieren/chinese-copywriting-guidelines

Reply:
As the size of the patch is huge, it might be helpful to split it into separate commits first:

(1) formatting-only-changes (spaces, quotes, ...) => either we want all of  them or none of them
(2) textual changes of existing translations => these will probably require most discussions
(3) new translations

@taotieren I have several scripts to process mapping files, so I guess it will take me ~1 hour to separate (1)+(2) from (3) automatically. Maybe I can even separate (1) from (2) automatically. If you think you can do it equally fast, please do yourself. Otherwise, I will do. Just let me know.

Reply:
It's better to use scripts. If not, I can do it manually.

Reply:
@mstrap I've already handled this version manually, and I hope to use scripts to handle this typesetting problem as well.

Reply:
@taotieren OK, I will split the commits as suggested, push to some branch and send you a pull request.

Reply:
@mstrap Thanks！

Reply:
@mstrap Can the translation file be generated into a file with *.pot or *.po extension? In this way, I can use Poedit to translate, and I can compare the original text with the translation better. The quality of translation can also be improved.

Reply:
@taotieren, I have sent a pull request now with split commits. When comparing your `patch-5` with my `split-translations` branch you will see only cleanup-related changes (in trailing spaces) *and* a change to `dlgSgPreferences.rbt"Working Tree"=工作树` which is caused by the `:*"Working Tree"=工作树` being applied on save.

Reply:
@taotieren, which minimum changes to the translation file would be necessary to have Poedit applicable? Note that current translation files use many SmartGit-specific tricks, like ":"-macros, various wildcard combinations, ...

Reply:
Thanks，There are some optimizations based on you.
`:*"Repositories"=版本库`
`:*"Repositories"=仓库`

Reply:
@taotieren, pull request #9 is a single commit again. I would suggest that you take my three commits instead and rebase your repository change on top of it (effectively making the pull requests 4 commits in size).

Reply:
@taotieren, please also change the commit author for my three commits to yourself. You can do this in the SmartGit Log, right-click the commit, *Edit Author*.